common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt

```diff
@@ -100,4 +112,26 @@ constructor(
             }
         }
     }
+
+    enum class TextColor {
+        Primary,
+        Secondary;
+
+        companion object {
+            fun from(type: Int): TextColor {
+                // same order as attrs-typography.xml
+                return when (type) {
+                    0 -> Primary
+                    else -> Secondary
+                }
+            }
+
+            fun getTextColorStateList(textColor: TextColor): Int {
+                return when (textColor) {
+                    Primary -> R.color.primary_text_color_selector
+                    Secondary -> R.color.secondary_text_color_selector
```

Comment:
Question: Do they have this new attribute defined in Figma so we can check it when implementing it?